src/cards/ScenarioNode/ScenarioNode.js

```diff
@@ -357,10 +388,20 @@ ScenarioNode.defaultProps = {
     created: 'Created',
     delete: 'Delete this scenario',
     redirect: 'Redirect to scenario view',
+    scenarioRename: {
+      title: 'Scenario name',
```

Comment:
`scenarioRename.title` is used in default value, but `scenarioRename.label` is used in comment and propType defined above.